### ISSUE TITLE
npm run build failed (both CMD and POWERSHELL)

### ISSUE THREAD

**Issue by JimKay1941** [hoppula/refire-forum#28]
Here is the cmd-prompt output: (I can provide the entire run and the logfile, if needed)

K:\GitHub\refire-forum>npm run build

> refire-forum@1.5.0 build K:\GitHub\refire-forum
> npm run clean && npm run build:webpack && npm run build:rules && npm run html

> refire-forum@1.5.0 clean K:\GitHub\refire-forum
> rimraf dist

> refire-forum@1.5.0 build:webpack K:\GitHub\refire-forum
> NODE_ENV=production webpack --config webpack.config.prod.js

'NODE_ENV' is not recognized as an internal or external command,
operable program or batch file.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! refire-forum@1.5.0 build:webpack: `NODE_ENV=production webpack --config webpack.config.prod.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the refire-forum@1.5.0 build:webpack script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\europ\AppData\Roaming\npm-cache\_logs\2019-11-25T17_58_06_520Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! refire-forum@1.5.0 build: `npm run clean && npm run build:webpack && npm run build:rules && npm run html`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the refire-forum@1.5.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\europ\AppData\Roaming\npm-cache\_logs\2019-11-25T17_58_06_571Z-debug.log

K:\GitHub\refire-forum>

**Comment by JimKay1941** [comment]
I have determined that the BABEL_ENV= syntax is invalid on windows 10 for both CMD and PowerShell. I edited all the places to be $env:NODE_ENV = 'xxxxxx' (for PowerShell).

I also noticed a few BABEL_ENV that needed to be updated.

Then I discovered that webpack needed to be installed. So I installed that.

Then I discovered I had to add node_modules/.bin/webpack to get it working.

There are far too many places where webpack in used to try editing everything, so I stopped.

It's a really nice forum system, but it may be too much work to get it running.
